src/Components/Config/configHooks.tsx

```diff
@@ -45,6 +45,15 @@ type OptionItem = {
   icon: IconItem;
 };
 
+function isValidJson(json: string): boolean {
+  try {
+    JSON.parse(json);
+    return true;
+  } catch {
+    return false;
+  }
+}
+
```

Comment:
I don't think this is used anywhere anymore.
```suggestion
```

Reply:
I've removed the function.